### ISSUE TITLE
Inflation of logger files

### ISSUE THREAD

**Issue by ttusar** [numbbo/coco-experiment#14]
Currently, the observer creates several files for each problem: `dat`, `tdat`, `rdat`, `mdat` (and `adat` in the bi-objective case).  However, `rdat` and `mdat` are often empty. 

Is this an issue? Should/can we do something about it?

**Comment by brockho** [comment]
I agree that it would be nicer to have (c)leaner output folders. The only place where to do something about it, is most likely the logger itself, maybe deleting the file if it turns out that no actual data was written? As @nikohansen said yesterday, the `.rdat` files are anyway of no use with the newest `exampleexperiment`...

**Comment by nikohansen** [comment]
I'd suggest to have a readme file which explains the file extensions and sits in the root output folder. This should ease the user experience. Otherwise, why even bother?
